status-im/status-react
# feature #3916 - added a few ropsten erc20 tokens, rearranged icon res…

fixes #3916

### Summary:

Adds a bunch of test tokens on Ropsten network:

```
HND
Handy Test Token
0x9e47fb3049f0d9c953f5428ce2e6c3a8321780bf

LXS
Lucky XS Test
0xf29d2dc0687d7d49f57d4a731ac8bfb6edc23473

ADI
Adi Test Token
0xd2a816110c1177478c7e644ae4853d8e80aaec35

WGN
Wagner Test Token
0x65c69bc258afa0906683f42e576b272a95c203dd

MDS
Modest Test Token
0x972b0570d9cd8b7c41aa8349f707ec7356daa825
```

User can get 1000 of each of those by sending exactly 0 ether to the appropriate contract address.

## Review notes

This PR also rearranges token assets to clearly separate test assets from any mainnet ones and thus avoid the confusion.


### Steps to test:
- Open Status
- Check if mainnet assets still look ok
- Switch to Ropsten and try out these


status: ready
